app/views/landing/index.slim

```diff
@@ -0,0 +1,31 @@
+doctype html
+html
+  = render 'header'
```

Comment:
You don't have to declare the doctype and the header for each view, it's all defined in the application.html.slim and then rendered in the 'yield' section.